BacktraceScreen.c

```diff
@@ -0,0 +1,383 @@
+/*
+htop - BacktraceScreen.h
+(C) 2023 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#include "BacktraceScreen.h"
+
+#if defined(HAVE_BACKTRACE)
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/wait.h>
+
+#include <ncursesw/curses.h>
```

Comment:
Always include ncurses via the `ProvideCurses.h` header …

```suggestion
#include "ProvideCurses.h"
```